pkg/operator/resourcesynccontroller/resourcesync_controller.go

```diff
@@ -126,7 +137,10 @@ func (c *ResourceSyncController) SyncSecret(destination, source ResourceLocation
 
 	c.syncRuleLock.Lock()
 	defer c.syncRuleLock.Unlock()
-	c.secretSyncRules[destination] = source
+	c.secretSyncRules[destination] = ResourceSourceLocation{
```

Comment:
then add a new map for the partial resources, ?

Reply:
I don't think that's necessary